utils/url.tsx

```diff
@@ -89,3 +93,41 @@ export function getScheme(url: string): string | null {
 
     return match && match[1];
 }
+
+function formattedError(id: string, message: string): React.ReactElement {
+    return (<span key={id}>
+        <FormattedMessage
+            id={id}
+            defaultMessage={message}
+        />
+        <br/>
+    </span>);
+}
+
+export function validateChannelUrl(url: string): React.ReactElement[] {
+    const errors: React.ReactElement[] = [];
+
+    const cleanedURL = cleanUpUrlable(url);
+    const urlMatched = url.match(/[a-z0-9]([-_\w]*)[a-z0-9]/);
+    if (cleanedURL !== url || !urlMatched || urlMatched[0] !== url || url.indexOf('__') > -1) {
+        if (url.length < 2) {
+            errors.push(formattedError(t('change_url.longer'), 'URL must be two or more characters.'));
```

Comment:
error strings for user still call it URL, maybe `channel's url name` or something along those lines will be less confusing? As I said I'm not concerned about code, but user experience. URL is the full thing, not just the name for it.

Reply:
I understand your point - `Slug` is technically the correct word for it that but I don't think this is the right place to do it.
If anything we would need to create new ticket (to be reviewed by UX potentially?)that modifies all existing references of `channel url` to something else but changing it in this one place while leaving it `URL` in all other places doesn't make sense to me. 

thoughts?

Reply:
sure, creating a new ticket to be reviewed by UX would be good engough :)